dnf.spec

```diff
@@ -21,7 +21,7 @@ URL:		https://github.com/rpm-software-management/dnf
 # cd dnf/package
 # ./archive
 # tarball will be generated in $HOME/rpmbuild/sources/
-Source0:    http://rpm-software-management.fedorapeople.org/dnf-%{version}.tar.gz
+Source0:    https://github.com/rpm-software-management/dnf/archive/%{name}-%{version}-1.tar.gz
 BuildArch:  noarch
```

Comment:
I don't think this will generalize the solution. Instead you may want to use it like this
%global buildno 1
at top of spec file and then
Source0: https://github.com/rpm-software-management/dnf/archive/%{name}-%{version}-%{buildno}.tar.gz